README.md

```diff
@@ -1,2 +1,87 @@
+
+For each _project group_ it is recommended that a “Project” is opened and actively maintained with high-level progress updates in the `srobo` GitHub - even the “nice to haves” - if there are good ideas being worked on which get parked here, at least there will be some documentation to pick up on.
+
+Examples of what may be in scope for SR2020: Python 3, new brain board, new IDE.
+
+### Project of significant impact: New brain board
+
+- A (group of) volunteer(s) decide(s) they’d like to work on finding a new single board computer which can serve as the brain board, and port/upgrade software to work on it.
+- New / Updated ticket in srobo/dev-tasks.
+- The volunteers develop a brief specification, including information like:
+    - Functional requirements (e.g. compatible with the rest of the kit, can be powered from Power Board, runs our API, camera isn’t made slower, form factor)
+    - Scope (e.g. excludes any changes to modular kit or API, independent of Python 3 project)
+    - Deliverables (e.g. Board, Software, Documentation for students and volunteers)
+    - Testing requirements (e.g. is demonstrated to run a defined piece of software on a test robot without fault, with a certain response time for libkoki markers, etc.)
+- The _Dev Team_ may also define some limits, such as:
+    - Cut-off date for deployment at SR2020
+    - Cost per unit limit
+    - Evidence shown that more than one board has been considered
+- The _Dev Team_ may consult the _Core Team_ or Trustees to assist them with requirements. For example, in the case of a Project that would have a high implementation cost, the _Dev Team_ would need to communicate the feasibility of it with the Trustees.
+- This specification is presented to and ratified by the _Dev Team_ (either on the _Dev Team_ mailing list or at a _Dev Team_ meeting)
+- Development work happens.
+- If changes to the specification are needed, these can be discussed with the _Dev Team_.
+- Development reaches a point where the project group is happy the work is done and is shippable.
+- _Dev Team_ reviews the equipment against the agreed specification and decides whether to recommend it to the _Core Team_.
+- The process is now passed onto the _Core Team_ who are responsible for deciding when and how to deploy it (e.g for SR 2020).
+    - If it meets the needs of the competition, it is the _Core Team_’s responsibility to ensure new kit is purchased, tested and deployed (although it’s likely to be the _project group_ who do this).
+    - If it doesn’t, the _Core Team_ must decide whether to proceed anyway at risk, or to continue with the existing kit and pause expansion.
+
+### Project of minor impact: Addition to the API (e.g. making an “R.ruggeduinos[0].PWM()” )
+
+As this is notionally a simpler thing, but still in scope as it impacts the competition cycle, the process can be simpler.
+
+- Developer wants to add the feature and contacts the appropriate _project group_ or creates one if it does not exist.
```

Comment:
What benefit does contacting an existing group bring? If there's an existing group rewriting the related code, and the shipment dates would match, then perhaps. But I suspect that level of project merging should be handled by the dev team rather than the volunteers themselves.
eg:
> "I want to add PWM to the ruggeduino in time for the next competition"
>
> "That's great! There's a team working on rewriting the ruggeduino firmware, which looks like it's going to be ready for the next competition, you should chat with them, see if they need any help!"

Reply:
Hence appropriate.

Reply:
The wording is still not quite right. 
```suggestion
- Developer wants to add the feature and is pointed in the direction of the appropriate _project group_ by the _Dev Team, or creates one if it does not exist.
```